### ISSUE TITLE
Source dropdown goes behind other cards on mobile

### ISSUE THREAD

**Issue by Sjorsa** [kalkih/mini-media-player#452]
Like [this](https://i.imgur.com/o9UELaS.png)

**Comment by TTLucian** [comment]
Also, if the card is at the bottom, source dropdown is not visible (goes out of the screen with no way to scroll)

![Untitled (Phone)](https://user-images.githubusercontent.com/1855276/108363360-819ac300-71fd-11eb-8ec5-4e96022a2336.jpg)

**Comment by kongo09** [comment]
Happens for me on desktop as well.

**Comment by kongo09** [comment]
Update: actually, the source dropdown works well for me, but the shortcuts dropdown goes into hiding.

I tried to understand the DOM and look for differences between the source and the shortcuts dropdown. Both have the z-index set to the same value with the same positioning. I don't understand why this happens.

**Comment by Sjorsa** [comment]
I just checked and it works as expected on mobile and desktop, should I leave the issue open because @TTLucian 's problem is not fixed yet?

**Comment by TTLucian** [comment]
Just checked. Source dropdown still behaves the same for me (just like in previous screenshot).

**Comment by kongo09** [comment]
Yes, please leave this open. It definitely does not work for me on both, mobile and desktop. If I can do anything to help solving this, e.g. providing some screens or configurations to test, please let me know.

**Comment by kongo09** [comment]
Crazy, I just wanted to make some screen shots to illustrate my situation - and now it works. I have spent an hour on this yesterday and it did totally not work but now it looks great. And the system really didn't change. So from my perspective, this can be closed.

![image](https://user-images.githubusercontent.com/640121/114894128-a6cb3c80-9e0e-11eb-8385-79be412f9aa3.png)
